content/en/dei/inclusive_culture_2021_grant.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Advancing an inclusive culture in the scientific Python ecosystem"
+draft: false
+---
+
+This joint NumPy, SciPy, Matplotlib, and Pandas project focuses on
+creating a DEI action plan and dedicated "Contributor Experience Lead" positions
+to identify, document, and implement practices to foster inclusive open-source communities.
+
+{{< page_gallery pages="inclusive_culture" >}}
+
+> We are happy to announce the Chan Zuckerberg Initiative has [awarded a grant](https://chanzuckerberg.com/newsroom/czi-awards-16-million-for-foundational-open-source-software-tools-essential-to-biomedicine/) to support the onboarding, inclusion, and retention of people from historically marginalized groups on scientific Python projects, and to structurally improve the community dynamics for NumPy, SciPy, Matplotlib, and Pandas.
+
+> As a part of [CZI's Essential Open Source Software for Science program](https://chanzuckerberg.com/eoss/), this [Diversity & Inclusion supplemental grant](https://cziscience.medium.com/advancing-diversity-and-inclusion-in-scientific-open-source-eaabe6a5488b) will support the creation of dedicated Contributor Experience Lead positions to identify, document, and implement practices to foster inclusive open-source communities. This project will be led by Melissa Mendonça (NumPy), with additional mentorship and guidance provided by Ralf Gommers (NumPy, SciPy), Hannah Aizenman and Thomas Caswell (Matplotlib), Matt Haberland (SciPy), and Joris Van den Bossche (Pandas).
+
+> This is an ambitious project aiming to discover and implement activities that should structurally improve the community dynamics of our projects. By establishing these new cross-project roles, we hope to introduce a new collaboration model to the Scientific Python communities, allowing community-building work within the ecosystem to be done more efficiently and with greater outcomes. We also expect to develop a clearer picture of what works and what doesn't in our projects to engage and retain new contributors, especially from historically underrepresented groups. Finally, we plan on producing detailed reports on the actions executed, explaining how they have impacted our projects in terms of representation and interaction with our communities.
+
+> The two-year project is expected to start by November 2021, and we are excited to see the results from this work! [You can read the full proposal here](https://figshare.com/articles/online_resource/Advancing_an_inclusive_culture_in_the_scientific_Python_ecosystem/16548063).
```

Comment:
Just a nit, but the several literal blocks renders kind of strangely in the html: https://208-142606827-gh.circle-artifacts.com/0/public/dei/inclusive_culture_2021_grant.html

It seems more like a collection of testimonals or something instead of a single quoted document

Reply:
How about something like this:

![2021-10-20_22:47:35](https://user-images.githubusercontent.com/45071/138218904-061689ae-b0a7-4605-b78a-f63ae42bc117.png)

We'd need to run this by numpy.org too, since they use quotes (although I think the dark background quotes look wonky there too).

Reply:
See https://github.com/scientific-python/scientific-python-hugo-theme/pull/32